zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -1739,6 +1739,12 @@ and [SASL authentication for ZooKeeper](https://cwiki.apache.org/confluence/disp
     Disabling it only recommended for testing purposes.
     Default: true
 
+* *ssl.clientHostnameVerification* and *ssl.quorum.clientHostnameVerification* :
+    (Java system properties: **zookeeper.ssl.clientHostnameVerification** and **zookeeper.ssl.quorum.clientHostnameVerification**)
+    **New in (INSERT VERSION HERE):**
+    Specifies whether the client's hostname verification is enabled in client and quorum TLS negotiation process.
```

Comment:
Please add that this setting will require server `hostnameVerification` setting to be true.

Reply:
2a7980dfc8980df7a831ac93be3d115db35cfe90